test/models.jl

```diff
@@ -0,0 +1,24 @@
+using Test, Random, AdvancedHMC
```

Comment:
@yebai I added a new test file call `models.jl` so we could add more typical models in the future.

Reply:
nice!